pkg/terminal/command.go

```diff
@@ -864,6 +869,17 @@ func parseArgs(args string) ([]string, error) {
 	return v[0], nil
 }
 
+// parseOptionalCount parses an optional count argument.
+func parseOptionalCount(arg string, unspecifiedValue int64) (val int64, err error) {
+	if len(arg) == 0 {
+		return unspecifiedValue, nil
+	}
+	if val, err = strconv.ParseInt(arg, 0, 64); err != nil {
```

Comment:
You can just `return strconv.ParseInt`, the value return parameter is ignored anyway if err is set.

Reply:
OK!